Eask

```diff
@@ -14,6 +14,7 @@
 
 (depends-on "emacs" "24.3")
 (depends-on "lsp-mode")
+(depends-on "haskell-mode")
```

Comment:
There is another `(depends-on "haskell-mode")` on line 25. You should the one in `(development  ..` scope.

Reply:
Make sense, thanks. Done.